integration-test/src/test/java/org/sagebionetworks/IT970UserProfileController.java

```diff
@@ -110,7 +112,12 @@ public void testFavoriteCrud() throws Exception {
 		PaginatedResults<EntityHeader> favs = synapse.getFavorites(Integer.MAX_VALUE, 0);
 		assertEquals(1, favs.getTotalNumberOfResults());
 		assertEquals(1, favs.getResults().size());
-		
+
+		// retrieve using sort params (sort order/direction are tested in the DAO)
+		favs = synapse.getFavorites(Integer.MAX_VALUE, 0, SortBy.NAME, SortDirection.ASC);
+		assertEquals(1, favs.getTotalNumberOfResults());
+		assertEquals(1, favs.getResults().size());
+
```

Comment:
As Marco suggested, I am calling the endpoint with and without the enum parameters. I'm not testing the sorting itself because the DAO integration test already does it.